### PR TITLE
Short article on git blame and ignoring it

### PR DESCRIPTION
**Pull request by ns-rse**
Had to do the masking of `git blame` for a project and wrote it up, the alias is a bonus and thought it worth describing how to use `git blame` at the command line.

### CODE REVIEW

File: _posts/2022-12-22-git-blame.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+git config --global alias.praise blame
+git praise -L1,30 git_blame.org
+# who alias
+git config --global alias.who blame
+git who -L1,30 git_blame.org
+```
+
+# Usage
+
+Git blame works on individual files and so requires a filename, there are a host of options, for example ~-e~ prints the
```
Comment (davidwyld) [inline _posts/2022-12-22-git-blame.md]: should these tildes be backticks? Might be a difference in the markdown renderer from where you first published this, kramdown renders these just as tildes in the text.

File: _posts/2022-12-22-git-blame.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+As of [2022-03-08](https://github.com/orgs/community/discussions/5033#discussioncomment-2318478) GitHub will also
+[ignore commits in the blame
+view](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view)
+that are listed in `.git-blame-ignore-revs` providing this file is in the root of your project folder.
+
+# Links
+
+## General
+
+* [Atlassian | git blame](https://www.atlassian.com/git/tutorials/inspecting-a-repository/git-blame)
```
Comment (davidwyld) [inline _posts/2022-12-22-git-blame.md]: something to do with the pipe character here is causing the link not to render correctly on my local deployment

**Review by davidwyld — CHANGES_REQUESTED** [review]
Looks great! Useful post. Couple of funky markdown gotchas in there.

File: _posts/2022-12-22-git-blame.md (hunk 1 above)
Comment (ns-rse) [reply _posts/2022-12-22-git-blame.md]: Yes they should be `~` is [org-mode](https://orgmode.org) ("orgdown"). Next time I'll force myself to use `org-export` to convert to Markdown and avoid such stupid mistakes, thanks again for the attention to detail, I'd not spotted it in Quarto.

File: _posts/2022-12-22-git-blame.md (hunk 2 above)
Comment (ns-rse) [reply _posts/2022-12-22-git-blame.md]: Thanks also, for checking this not something Quarto complained about. Hadn't thought to replicate the Ruby envinronment locally.

**Review by davidwyld — APPROVED** [review]
:+1:

**Pull request merged**